ui/analyse/src/study/commentForm.ts

```diff
@@ -112,10 +112,7 @@ export function view(root: AnalyseCtrl): VNode {
             insert(vnode) {
               setupTextarea(vnode);
               const el = vnode.elm as HTMLInputElement;
-              function onChange() {
-                setTimeout(() => ctrl.submit(el.value), 50);
-              }
-              el.onkeyup = el.onpaste = onChange;
+              el.oninput = () => setTimeout(() => ctrl.submit(el.value), 50);
```

Comment:
Also, not sure I understand this timeout? Would expect a throttle instead. Is setTimeout doing something like that here? Or is there another reason for it?

Reply:
I guess it delays `submit` a bit to increase the chances of the textarea value being up to date.